doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -0,0 +1,1016 @@
+   ---
+      • Create application test-app
+      • Generate application files
+      • Initialize application git repository
+      • Application "test-app" created successfully
+
+In the ``test-app`` directory, we get the app created from a template:
+
+.. code-block:: console
+
+   $ ls test-app
+   ---
+   ...
+
+   instances.yaml
+   test-app-scm-1.rockspec
+   ...
+
+The app is fully functional and can respond to the HTTP GET request ``/hello``.
+
+.. NOTE::
+
+   Check the cartridge version in *test-app-scm-1.rockspec*:
+
+   .. code:: lua
+
+      dependencies = {
+          ...
+          'cartridge == 2.3.0-1',
+          ...
+      }
+
+The version of Cartridge must be **>= 2.3.0**. Starting from this version,
+Cartridge waits for an instance to become available on its DNS address during
+the instance start. This is required for correct operations in Kubernetes. For
+versions below 2.3.0, an application must be customized independently.
+See the
+`example <https://github.com/tarantool/tarantool-operator/blob/master/examples/kv/key-value-store/init.lua#L27-L71>`_
+of how to do this.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Building application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's create a docker image using ``cartridge-cli``:
+
+.. code-block:: console
+
+   $ cartridge pack docker --tag vanyarock01/test-app:0.1.0-0-g68f6117
+   ---
+   ...
+   Running in 0ffbd57a0edf
+   Removing intermediate container 0ffbd57a0edf
+    ---> aceef7a3be63
+    ---> aceef7a3be63
+   Successfully built aceef7a3be63
+   Successfully tagged test-app:0.1.0-0-g68f6117
+      • Created result image test-app:0.1.0-0-g68f6117
+      • Application was successfully packed
+
+Upload the image to a registry:
+
+.. code-block:: console
+
+   $ docker push vanyarock01/test-app:0.1.0-0-g68f6117
+   ---
+   The push refers to repository [docker.io/vanyarock01/test-app]
+   b327b35afe0a: Pushed
+   de30ed3f758d: Pushed
+   3c8808fbd85d: Pushed
+   291f6e44771a: Pushed
+   0.1.0-0-g275baa8: digest: sha256:5b3b92a615b34c7f132e72e2d61f692cf2091ca28be27bbbfed98106398d1c19 size: 1160
+
+.. NOTE::
+
+   You must be logged in via ``docker login`` and have access rights
+   to the target registry.
+
+.. _cartridge_kubernetes_creating_a_kubernetes_cluster:
+
+--------------------------------------------------------------------------------
+Creating Kubernetes cluster
+--------------------------------------------------------------------------------
+
+If you have a ready-made cluster in the cloud, you can use it. If not, we
+suggest two ways of how to create a local cluster:
+
+* using :ref:`minikube <cartridge_kubernetes_creating_cluster_minikube>`
+* using :ref:`kind <cartridge_kubernetes_creating_cluster_kind>`.
+
+.. _cartridge_kubernetes_creating_cluster_minikube:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using *minikube*
```

Comment:
Here the name “minicube” is formatted with *italic*, just above it was plain text and on line 52 it was **bold**. This looks inconsistent. Not sure if we already have an established rule of formatting app names, but we probably should have it.